package.json

```diff
@@ -3,24 +3,20 @@
     "scripts": {
         "install-all": "npm install yarn --no-save && yarn",
         "i": "yarn",
-
```

Comment:
Just curious: did you remove these new lines or did yarn remove them?

Reply:
ow, sorry I didn't notice. I just touched the mdocs version number (and the example at the end) and ran `npm install` (usually I run `yarn` in reframe but for some reason this time I've run npm). I'll revert all changes to this file except the mdocs version number

Reply:
Ah ok so npm removed these lines. I'm not sure when `package.json` formating is kept and when not